src/erlpass.erl

```diff
@@ -5,12 +5,14 @@
 -export([hash/1, hash/2, match/2, change/3, change/4]).
 -define(DEFAULT_WORK_FACTOR, 12).
 
-%% @type password() = iodata(). A password, supports valid unicode.
 -type password() :: iodata().
-%% @type work_factor() = 4..31. Work factor of the bcrypt algorithm
+%% A password, supports valid unicode.
```

Comment:
does the documentation really go below the type it's documenting?

Reply:
> does the documentation really go below the type it's documenting?

Yes. The description and examples are [here](https://www.erlang.org/doc/apps/edoc/chapter.html#type-definitions).

For instance:

>    %% @type thing(A) = {thong, A}
>    %%           A = term().
 >   %%   A kind of wrapper type thingy.

Reply:
[Preview](https://disk.yandex.ru/i/_T5-shsGSMAsTg)